tutorials/frontend/build_gcn.py

```diff
@@ -240,43 +240,42 @@ def load_dataset(dataset="cora"):
 ######################################################################
 # Compile and run
 # ------------------
-# We achieved 6.5x speedup for this dataset against dgl given the same model parameters.
-# Output numerical difference < 10e-4 %.
 #
 # DGL version: https://github.com/dmlc/dgl/blob/master/examples/mxnet/gcn/gcn.py
-from tvm.contrib import graph_runtime
-import time
 
-# Set up weights. You can modify this part and use your own trained weights.
-params['in_weight'] = np.ones((input_dim, hidden_dim), dtype='float32')
-params['out_weight'] = np.ones((hidden_dim, num_classes), dtype='float32')
-for i in range(num_hidden):
-    params["%s_weight"%(str(i))] = np.ones((hidden_dim, hidden_dim), dtype='float32')
+# Download pretrained GCN model
+model_url = "https://homes.cs.washington.edu/~cyulin/media/gcn_%s.pickle"%(dataset)
```

Comment:
How was this pickle file generated? Was it trained manually?

Reply:
It's trained with this example DGL pytorch code
https://github.com/dmlc/dgl/blob/master/examples/pytorch/gcn/train.py
I can add the source to the comment.

Reply:
Yes, that would be good to make the tutorial easy to reproduce/extend